contributors/index.js

```diff
@@ -281,6 +281,14 @@ module.exports.list = [
     tech_stack: ["C++", "Python", "JavaScript", "React"],
     github_username: "upasanadeka",
     twitter_username: "upasana_deka"
+  },
+  {
+    name: "Siddhant",
+    country: "India",
+    role: "To be Filled",
```

Comment:
hi @FuzzySid . Can you fill in the "role" property with what type of developer you are or want to be !